app/src/main/java/org/gdg/frisbee/android/eventseries/GdgEventListFragment.java

```diff
@@ -37,58 +41,33 @@ EventAdapter createEventAdapter() {
     }
 
     @Override
-    void fetchEvents() {
-        final DateTime now = DateTime.now();
-        DateTime start = now.minusMonths(2).dayOfMonth().withMinimumValue();
```

Comment:
Here we were giving the `start` date to the API which was about 2 months before the current date time. Where did this behavior go?